src/main/resources/org/jpeek/metrics/LCOM4.xsl

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@llorllale Is the PDF necessary, should it be in the repo? Wouldn't a link to it in the README be better?

Reply:
@amihaiemil I don't know why but we're always adding the PDFs to the repo. I do agree that we shouldn't be committing these binaries. Perhaps just attaching them to the ticket should suffice. Not my call.